politeiawww/cmd/politeiavoter/config.go

```diff
@@ -72,6 +72,7 @@ type config struct {
 	Profile          string `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65536"`
 	DebugLevel       string `short:"d" long:"debuglevel" description:"Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set the log level for individual subsystems -- Use show to list available subsystems"`
 	Version          string
+	ListCommands     bool   `short:"l" long:"listcommands" description:"List available commands"`
```

Comment:
dcrctl has this as the second option.
```
Application Options:
  -V, --version                     Display version information and exit
  -l, --listcommands                List all of the supported commands and exit
 ```

politeiavoter currently has it burried in the options list, making it hard to find.  I would match the `dcrctl` behavior.